main.py

```diff
@@ -60,5 +63,26 @@ def submitNote():
         # Unsure if this would work. Directory is referenced, but seems redundant.
         return (qnTitle + '\n' + qnContent+ dateMessage)
 
-    return app
+    @app.route('/fileread', methods=['GET'])
```

Comment:
These are Scott's changes I believe, but since this wasn't part of your ticket you should just go ahead and discard all of your tracked changes to this whole file. You might have to do something like revert the branch or delete the PR and resubmit but since you weren't making any changes to the server code, we ought not to include an extra revision if we don't need to.